slick-codegen/src/main/scala/slick/codegen/AbstractGenerator.scala

```diff
@@ -91,7 +91,12 @@ abstract class AbstractGenerator[Code,TermName,TypeName](model: m.Model)
     /** Indicates whether auto increment columns should be put last and made an Option with a None default.
         Please set to !hlistEnabled for switching this on.
         @group Basic customization overrides */
+    @deprecated("Use autoIncLast and ColumnDef.asOption instead.", "3.2.0")
```

Comment:
minor enhancement. let's be more precise: "Instead set autoIncLast = true and Column.asOption = autoInc"